<launch>
    <env name="ROSCONSOLE_CONFIG_FILE" value="$(find exploration)/cfg/custom_rosconsole.config"/>

    <arg name="robot_name" default="robot1"/>
    <arg name="map_frame_id" default="$(arg robot_name)/map"/>

     <group ns="$(arg robot_name)">
        <node pkg="exploration" type="movement_test" name="movement_test" output="screen">
            <remap from="velocity" to="/$(arg robot_name)/mobile_base/commands/velocity"/>
            <remap from="global_costmap" to="/$(arg robot_name)/move_base/global_costmap/costmap"/>

            <rosparam file="$(find exploration)/param/movement_params.yaml" command="load"/>
            <param name="movement_costmap/global_frame" value="$(arg map_frame_id)"/>
            <param name="movement_costmap/robot_base_frame" value="$(arg robot_name)/base_footprint"/>
         </node>

         <include file="$(find exploration)/launch/includes/exploration_support.launch.xml">
            <arg name="frontier_detection" value="false"/>
            <arg name="branch_detection" value="false"/>
            <arg name="map_fill" value="false"/>
        </include>
    </group>
</launch> 